tests/lax_numpy_test.py

```diff
@@ -1752,6 +1752,27 @@ def testTrimZeros(self, a_shape, dtype, trim):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}_rank{}".format(
+        jtu.format_shape_dtype_string(a_shape, dtype), rank),
+       "dtype": dtype, "a_shape": a_shape, "rank": rank}
+      for rank in (1, 2)
+      for dtype in default_dtypes
+      for a_shape in one_dim_array_shapes))
+  def testPoly(self, a_shape, dtype, rank):
+    if np.mintypecode(dtype(0).dtype.char) == 'd' and not config.x64_enabled:
+      self.skipTest(f"Only run {dtype} testcase when float64 is enabled.")
+    elif dtype == np.float32 and not config.x64_enabled and rank == 2:
+      self.skipTest(f"Only run {dtype} testcase with rank 2 when "
+                    "float64 is enabled (due to the need for complex64).")
```

Comment:
`complex64` is available with X64 disabled. You should be able to remove this skip (I suspect the reason you saw it fail is because of the warning addressed in the comment above)

Reply:
Yes, that sounds right. I removed this skip in https://github.com/google/jax/commit/d7a0190b545475e4140feebf411082c4b2b10245.